modules/statics/src/coins.ts

```diff
@@ -619,8 +618,17 @@ export const coins = CoinMap.fromCoins([
   erc20('zrx', '0x Token', 18, '0xe41d2489571d322189246dafa5ebde1f4699f498', UnderlyingAsset.ZRX),
   erc20('zusd', 'Z.com Usd', 6, '0xc56c2b7e71b54d38aab6d52e94a04cbfa8f604fa', UnderlyingAsset.ZUSD),
   celoToken('cusd', 'Celo USD', 18, '0x765de816845861e75a25fca122bb6898b8b1282a', UnderlyingAsset.CUSD),
-  ofcerc20('ofcusdt', 'Tether', 6, UnderlyingAsset.USDT),
   ofcerc20('ofcusdc', 'USD Coin', 6, UnderlyingAsset.USDC),
+  ofcerc20('aave', 'Aave', 18, UnderlyingAsset.AAVE),
```

Comment:
The coin tickers need to be globally unique so that's causing the tests to fail. Can we continue using the `ofc` prefix on these?